app/components/SelectWallet.tsx

```diff
@@ -25,22 +27,29 @@ interface SelectWalletProps {
 export function SelectWallet({ isProductionMode }: SelectWalletProps) {
 	const { isLoading, isWalletConnected } = useContext(WalletContext);
 	const { connectWallet } = useXverseConnect();
+	const location = useLocation();
+	const currentPath = location.pathname;
+
+	// hide wallet based on routes
+	const shouldShowBitcoinWallet =
+		wallet?.routes?.[currentPath as keyof typeof wallet.routes]?.bitcoin ?? true;
```

Comment:
If we use typescript in the config file then this will be simpler - see comment below

Reply:
Done